rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :person, :date
+  attr_reader :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    person.rentals << self
+    book.rentals << self
+  end
+end
```

Comment:
- [ ] Please your app still generate an error when a student tries to `rent` a book.
Please fix that as early mentioned by the previous code reviewer.
![rentalerror](https://user-images.githubusercontent.com/55596970/224473237-77284ff2-1573-48a9-9159-7d04f153af3c.png)